src/Twig/Extension/PaginationRuntime.php

```diff
@@ -9,11 +9,11 @@
 
 final class PaginationRuntime implements RuntimeExtensionInterface
 {
-    private Processor $processor;
-
-    public function __construct(Processor $processor)
-    {
-        $this->processor = $processor;
+    public function __construct(
+        private readonly Processor $processor,
+        private readonly string $pageName = 'page',
+        private readonly bool $skipFirstPageLink = false,
```

Comment:
I'm not so fond of the naming of this parameter as it can be misinterpreted.
```suggestion
        private readonly bool $includePageInFIrstPageLink = true,
```